shared/src/email/emailTemplatesByName.ts

```diff
@@ -1067,8 +1067,44 @@ Ne tardez pas : répondez lui directement en utilisant le bouton ci-dessous : `,
         subContent: defaultSignature(internshipKind),
       }),
     },
+    AGENCY_ASSESSMENT_NOTIFICATION: {
+      niceName: "Prescripteurs - Lien de création du bilan",
+      tags: ["notification prescripteur fin de l’immersion"],
+      createEmailVariables: ({
+        agencyLogoUrl,
+        assessmentCreationLink,
+        beneficiaryFirstName,
+        beneficiaryLastName,
+        businessName,
+        conventionId,
+        internshipKind,
+      }) => ({
+        subject: `Immersion Facilitée - Bilan non complété pour l'immersion de ${beneficiaryFirstName} ${beneficiaryLastName}`,
+        greetings: greetingsWithConventionId(conventionId),
+        content: `
+      ${
+        internshipKind === "immersion"
+          ? "L'immersion professionnelle"
+          : "Le mini stage"
+      } de ${beneficiaryFirstName} ${beneficiaryLastName} au sein de l’entreprise ${businessName} est en passe de s'achever.
+
+      Nous venons d’envoyer le bilan de l’immersion à l’entreprise. Vous pouvez, si vous le souhaitez, les contacter par téléphone pour les accompagner dans la saisie du bilan.
+      `,
+        buttons: [
+          {
+            label: "Formulaire de bilan",
+            url: assessmentCreationLink,
+          },
+        ],
+        subContent: `
+      Ces informations sont importantes pour la suite du parcours professionnel de BENEFICIARY_FIRST_NAME BENEFICIARY_LAST_NAME.    
+      ${defaultSignature(internshipKind)}
+      `,
+        agencyLogoUrl,
+      }),
+    },
     ESTABLISHMENT_ASSESSMENT_NOTIFICATION: {
-      niceName: "Convention - Lien de creation du bilan",
+      niceName: "Convention - Lien de creation du bilan pour l'entreprise",
```

Comment:
tu penses quoi de créer une catégorie `Bilan` ? Qu'on pourrait réutiliser pour le mail d'au dessus également.

```suggestion
      niceName: "Bilan - Lien de creation du bilan pour l'entreprise",
```

Reply:
Yep j'ai refait les catégories ainsi que les KINDs des emails. Ca inclus maintenant une migration pour changer les notifs emails passées.